examples/platform/nxp/common/app_task/source/AppTaskBase.cpp

```diff
@@ -89,20 +89,11 @@
 #include "OTARequestorInitiator.h"
 #endif
 
-#if CONFIG_CHIP_TEST_EVENT && CHIP_DEVICE_CONFIG_ENABLE_OTA_REQUESTOR
-#include <app/clusters/ota-requestor/OTATestEventTriggerDelegate.h>
-#endif
-
 #ifdef ENABLE_CHIP_SHELL
 #include <lib/shell/commands/WiFi.h>
 #endif
 
-#ifdef SMOKE_CO_ALARM
-#include <app/TestEventTriggerDelegate.h>
-#include <app/clusters/smoke-co-alarm-server/SmokeCOTestEventTriggerHandler.h>
-#endif
-
-#if CHIP_CONFIG_ENABLE_ICD_SERVER
+#ifdef CONFIG_CHIP_REGISTER_SIMPLE_TEST_EVENT_TRIGGER_DELEGATE
 #include <app/TestEventTriggerDelegate.h>
```

Comment:
Can you add the include to #include <app/clusters/ota-requestor/OTATestEventTriggerHandler.h> in case CONFIG_CHIP_REGISTER_SIMPLE_TEST_EVENT_TRIGGER_DELEGATE and CHIP_DEVICE_CONFIG_ENABLE_OTA_REQUESTOR are enabled ?